pyiron/dft/job/generic.py

```diff
@@ -78,13 +78,12 @@ def exchange_correlation_functional(self, val):
     def get_k_mesh_by_cell(self, cell=None, kpoints_per_angstrom=1):
         if cell is None:
             cell = self.structure.cell
-        latlens = [np.linalg.norm(lat) for lat in cell]
-        kmesh = np.rint(
-            np.array([2 * np.pi / ll for ll in latlens]) * kpoints_per_angstrom
-        )
-        if kmesh.min() <= 0:
-            warnings.warn("kpoint per angstrom too low")
-        return [int(k) for k in kmesh]
+        return get_k_mesh_by_cell(cell, kpoints_per_angstrom)
+
+    def get_k_mesh_by_density(self, cell=None, kmesh_density_per_inverse_angstrom=1):
```

Comment:
In most of the cases, what we need is the k mesh of the current cell, so it's not really intuitive to me that I have to put cell before the k mesh density. I'd rather change the order and remove the mysterious `1` in the argument.

Reply:
> In most of the cases, what we need is the k mesh of the current cell, so it's not really intuitive to me that I have to put cell before the k mesh density. I'd rather change the order and remove the mysterious `1` in the argument.

I'm mainly waiting for this one. Other than that, I'm fine with merging it